services/wiki/wiki.go

```diff
@@ -81,6 +86,36 @@ func InitWiki(repo *models.Repository) error {
 	return nil
 }
 
+// prepareWikiFileName try to find a suitable file path with file name by the given raw wiki name.
+// return: existence, prepared file path with name, error
+func prepareWikiFileName(gitRepo *git.Repository, wikiName string) (bool, string, error) {
+	// Pass in the raw unchanged name
+	// Add ".md" suffix if not there.
```

Comment:
```suggestion
```

I'd prefer to remove this comment and don't do the check since Markdown is the only supported markup language in Wiki. User doesn't need to know or manually specify the file extension here.